tests/test_convolved.py

```diff
@@ -243,7 +255,7 @@ def setup_module(module, backend="virtualDevice"):
     lsst.utils.tests.init()
     try:
         afwDisplay.setDefaultBackend(backend)
-    except:
+    except:  # noqa E722: want to catch any and all errors
```

Comment:
Please catch `BaseException`. It is no longer standard practice in python3 to have a bare except.